include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -1744,13 +1755,18 @@ __pattern_reverse_copy(_ExecutionPolicy&& __exec, _RandomAccessIterator1 __first
                        _RandomAccessIterator2 __d_first, _IsVector __is_vector, /*is_parallel=*/::std::true_type)
 {
     auto __len = __last - __first;
-    __par_backend::__parallel_for(::std::forward<_ExecutionPolicy>(__exec), __first, __last,
-                                  [__is_vector, __first, __len, __d_first](_RandomAccessIterator1 __inner_first,
-                                                                           _RandomAccessIterator1 __inner_last) {
-                                      __internal::__brick_reverse_copy(__inner_first, __inner_last,
-                                                                       __d_first + (__len - (__inner_last - __first)),
-                                                                       __is_vector);
-                                  });
+
+    if (__len > 0)
```

Comment:
I would suggest return on trivial case like in other patterns, f.e. https://github.com/oneapi-src/oneDPL/pull/1442/files#diff-39feab6040f4974b495666d46a63d64a1fcbe9fcb836d5bad9d645d1a6129184L1996

In that case we dont need to have indentations for many lines where a parallel backend pattern is called.